kernel/packages/config/index.ts

```diff
@@ -289,6 +289,8 @@ export function getServerConfigurations() {
   const metaFeatureFlagsBaseUrl = `https://feature-flags.decentraland.${notToday}/explorer.json`
   const ASSET_BUNDLES_DOMAIN = qs.ASSET_BUNDLES_DOMAIN || `content-assets-as-bundle.decentraland.${TLDDefault}`
 
+  const QUESTS_SERVER_URL = qs.QUESTS_SERVER_URL ?? `https://quests-api.decentraland.${notToday}`
```

Comment:
I think that `notToday` would be either `org` or `zone`, not `io`. Is that ok? Would it still work?

Reply:
Oops...